swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/generator/core/processor/annotation/SwaggerDefinitionProcessor.java

```diff
@@ -74,20 +74,27 @@ private void convertInfo(io.swagger.annotations.Info infoAnnotation, Swagger swa
   private License convertLicense(io.swagger.annotations.License licenseAnnotation) {
     License license = new License();
 
-    license.setName(licenseAnnotation.name());
-    license.setUrl(licenseAnnotation.url());
-
-    return license;
+    license.setName(getStringValue(licenseAnnotation.name()));
+    license.setUrl(getStringValue(licenseAnnotation.url()));
+    if (null == license.getName() && null == license.getUrl()) {
```

Comment:
How about the license name is null but the url is not null?
It's better to use StringUtils.isEmpty directly instead of getStringValue.

Reply:
This method is for empty value filter purpose. But this is no need to do as wujimin commented. So these have been removed.